libs/components/lookup/src/lib/modules/country-field/country-field.component.spec.ts

```diff
@@ -557,7 +557,6 @@ describe('Country Field Component', () => {
           iso2: 'au',
           dialCode: '61',
           priority: 0,
-          areaCodes: null,
```

Comment:
`areaCodes` is an internal property defined as `string[] | undefined`. The fact that we're checking if it's `null` in the test is odd; I modified the code to not set the property if the area codes do not exist.